http/src/lib.rs

```diff
@@ -9,14 +9,13 @@
     unsafe_code,
     unused
 )]
-#![doc = include_str!("../README.md")]
 #![allow(
-    clippy::missing_errors_doc,
     clippy::module_name_repetitions,
     clippy::must_use_candidate,
-    clippy::semicolon_if_nothing_returned,
-    clippy::unnecessary_wraps
+    clippy::unnecessary_wraps,
+    clippy::used_underscore_binding
 )]
+#![doc = include_str!("../README.md")]
```

Comment:
why reorder this?

Reply:
seems to me that deny only needs to be above allow due to how the directives work, otherwise we can alphabetize it